clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -2198,6 +2403,14 @@ mlir::Value CIRGenFunction::buildLoadOfScalar(Address Addr, bool Volatile,
                                               QualType Ty, SourceLocation Loc,
                                               LValueBaseInfo BaseInfo,
                                               bool isNontemporal) {
+  return buildLoadOfScalar(Addr, Volatile, Ty, getLoc(Loc), BaseInfo,
+                           isNontemporal);
+}
+
+mlir::Value CIRGenFunction::buildLoadOfScalar(Address Addr, bool Volatile,
+                                              QualType Ty, mlir::Location Loc,
+                                              LValueBaseInfo BaseInfo,
+                                              bool isNontemporal) {
```

Comment:
Same review as before: could we fetch the required `mlir::Location` from `SourceLocation`?